src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1 +1,1433 @@
-UPDATE tab_populations SET display_name='East Asian (Tanaka, 1996)', description='East Asian (Tanaka, 1996)' WHERE population='Asian_Tanaka_1996';
+DROP TABLE tab_compound_process_parameter_mapping; -- due to schema mismatch
+CREATE TABLE [tab_compound_process_parameter_mapping](

```

Comment:
This is because I deleted the garbage columns (sometimes you find really funny things)
![grafik](https://user-images.githubusercontent.com/25061876/182362270-1ad2a7df-83cc-4324-aebd-ae20dacb6e4f.png)

Reply:
lol